src/routes/(app)/checkout/+page.svelte

```diff
@@ -70,7 +70,9 @@
 			}
 
 			input.value = input.value.trim();
```

Comment:
```suggestion
			input.value = trimPrefix(input.value.trim(), "nostr:");
```

`import { trimPrefix } from '$lib/utils/trimPrefix';`